spdockit-spo/spo-snapshots.md

```diff
@@ -38,5 +40,22 @@ Here is what you need to do:
 
 1. Click Next and the loading will start. Wait for the SPDocKit wizard to finish, then SharePoint [farm settings](#internal/get-to-know-spdockit/farm-explorer-screen/farm-explorer-reports) and [permission reports](#internal/get-to-know-spdockit/permissions-reports-screen) will be ready for use!  
 
-    Also, make sure to check our [permission management features](#internal/permission-management/manage-permissions-ribbon-actions) located in __Permissions Explorer__ tab.  
-    Please note that Permission management buttons are enabled only in __live mode__ of Permissions Explorer. There is no need for a snapshot in this case. 
+
+## Automatic Snapshot
+
+SPDocKit Service allows you to configure a schedule to create automatic SharePoint Online snapshots. Once configured, the SPDocKit Service can save your farm settings and send report subscriptions and alerts to selected users, document libraries, or network locations. Click [here](internal/how-to/subscriptions-and-alerts/create-new-subscription) for more information on how to do this.  
+
+It is not required to have Local Admin privilages for a service account. It can be any account, even Local System. See [here](https://msdn.microsoft.com/en-us/library/windows/desktop/ms684190(v=vs.85).aspx) for more information on the Local System account. While setting up the service account via configuration wizard, simply choose what is more convenient to you. 
+
+__Please note:__ Local System account can be used only for SPDocKit service (not possible for SPDocKit Insights Service) if one of the folowing conditions are met:
+* You are using an embedded database.
+* You are using SQL Authentication to connect to your database.
+* You are using a local SQL Server.  
```

Comment:
I believe that this is incorrect and we should expand our CW logic to include remote SQL servers. Tried it and it works, it's just that the account that needs to be given sql permissions should be domain\computerName$

Reply:
This is correct with the current implementation I believe? 
I agree that we should expand the CW logic, and if possible in the process avoid making the user know what the Local System account looks like or making him learn that himself. But that's a discussion to be had at a review or planning? (create a PBI if one does not already exist?)